ocw/lib/azure.py

```diff
@@ -206,7 +206,7 @@ def cleanup_images_from_rg(self, keep_images):
                     if self.dry_run:
                         self.log_info("Deletion of image {} skipped due to dry run mode", item.name)
                     else:
-                        self.compute_mgmt_client().images.delete(self.__resource_group, item.name)
+                        self.compute_mgmt_client().images.begin_delete(self.__resource_group, item.name)
```

Comment:
how this appears here ?  Looks like some mess in your commits tree , this change already in master  https://github.com/SUSE/pcw/commit/e6143d526c033ef19e33516e026981a41ecba5e7